### PR TITLE
configure sonar webhook, jenkins plugin

### PR DESCRIPTION
**Pull request by mrparkers**
depends on https://github.com/jdamata/terraform-provider-sonarqube/pull/38

### CODE REVIEW

File: tf/k8s/main.tf [hunk 1]
```diff
@@ -11,14 +11,18 @@ terraform {
...
       source  = "hashicorp/random"
       version = "3.0.1"
     }
-    harbor = {
+    harbor    = {
       source  = "liatrio/harbor"
       version = "0.3.4"
     }
+    sonarqube = {
+      source  = "github.com/jdamata/sonarqube"
```
Comment (mrparkers) [inline tf/k8s/main.tf:23]: this needs to be changed when the previously mentioned PR is merged and a new release is cut

File: tf/modules/sonarqube-config/versions.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.13.1"
+  required_providers {
+    sonarqube = {
+      source  = "github.com/jdamata/sonarqube"
```
Comment (mrparkers) [inline tf/modules/sonarqube-config/versions.tf:5]: same here

File: tf/modules/rode/policies/sonarqube.rego [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+violations[result] {
+	finished := sonar_scan_finished
+	result := {
+		"pass": count(finished) >= 1,
+		"id": "sonar_scan_completed",
+		"name": "SonarQube Analysis Finished",
+		"description": "Verify SonarQube analysis finished successfully",
+		"message": sprintf("SonarQube analysis completed at %v", [finished]),
```
Comment (alexashley) [inline tf/modules/rode/policies/sonarqube.rego:43]: We may want to stay away from formatted messages with values that don't have defaults. If the scan never finished, the output is a little wonky: https://github.com/rode/rode/issues/82

File: tf/modules/rode/policies/sonarqube.rego [hunk 4]
```diff
@@ -0,0 +1,45 @@
+package rode.demo.sonar1
```
Comment (alexashley) [inline tf/modules/rode/policies/sonarqube.rego:1]: Is there a Sonar2?

**Review by alexashley — APPROVED** [review]

File: tf/modules/rode/policies/sonarqube.rego (hunk 3 above)
Comment (mrparkers) [reply tf/modules/rode/policies/sonarqube.rego:43]: I'll probably take a look at this in another PR.  gotta familiarize myself more with rego first

File: tf/modules/rode/policies/sonarqube.rego (hunk 4 above)
Comment (mrparkers) [reply tf/modules/rode/policies/sonarqube.rego]: oops 😄

**Pull request merged**